### ISSUE TITLE
sd_isAnimated is always return YES

### ISSUE THREAD

**Issue by dhn123257** [SDWebImage/SDWebImage#3345]
### New Issue Checklist

* [YES] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [YES] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [YES] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

 sd_isAnimated is always return YES . So When I use SDAniamtedImageView to load static image,  I can't use this interface to distinguish between animated image and static image

**Comment by dreampiggy** [comment]
From v5.12+, that `sd_isAnimated` always return YES for both `_UIAnimatedImage` as well as `SDAnimatedImage`.

If you really want to distinguish these, use `[image isKindOfClass:SDAnimatedImage.class]` check

**Comment by dreampiggy** [comment]
That `sd_isAnimated` change for symatic is for most simple use cases, previouslly it return `NO` for `SDAnimatedImage` will cause a little misunderstanding like `Why my GIF/WebP image is animating, but the isAnimated is NO ?`
